.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+name: collect logs from runner
+
+on:
+  schedule:
+    - cron: 50 * * * *
+
+jobs:
+  collect_logs:
+    runs-on: self-hosted
+    steps:
+      - name: fetch.py for master
+        run: |
+          cd /mnt/storage/multivac
```

Comment:
take a look at `working_directory` param, it looks much better than `cd`
example: https://github.com/tarantool/infra/blob/master/.github/workflows/setup_runner.yml#L81
doc: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-set-the-default-shell-and-working-directory
and 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun